imports/plugins/included/shippo/lib/collections/schemas/shippo.js

```diff
@@ -0,0 +1,13 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+import { PackageConfig } from "/lib/collections/schemas/registry";
+
+export const ShippoPackageConfig = new SimpleSchema([
+  PackageConfig, {
+    "settings.apiKey": {
+      type: String,
+      label: "API Key",
+      min: 10,
+      optional: true
```

Comment:
Leaving it blank means it's not fully configured, and shippo will effectively be disabled. When an API key is entered then it's (really) enabled.